stripe/service.go

```diff
@@ -340,17 +381,34 @@ func processProductToPlan(planID uint64, product *stripeapi.Product) (*db.Plan,
 		return nil, err
 	}
 
-	return &db.Plan{
+	plan := &db.Plan{
 		ID:            planID,
 		Name:          product.Name,
-		StartingPrice: product.DefaultPrice.UnitAmount,
 		StripeID:      product.ID,
-		StripePriceID: product.DefaultPrice.ID,
 		Default:       isDefaultPlan(product),
 		Organization:  organizationData,
+		FreeTrialDays: s.config.FreeTrialDays,
```

Comment:
this is mixing sources of truth, why not define FreeTrialDays in stripe? 

then for example
```go
	freeTrialDays, err := extractPlanMetadata[int64](product.Metadata["freeTrialDays"])
	if err != nil {
		return nil, err
	}
```